coloring.dylan

```diff
@@ -0,0 +1,39 @@
+Module: %testworks
+Synopsis: Utilities and code that needs to be loaded early.
+Copyright: Original Code is Copyright (c) 1995-2004 Functional Objects, Inc.
+           All rights reserved.
+License: See License.txt in this distribution for details.
+Warranty: Distributed WITHOUT WARRANTY OF ANY KIND
+
+
+define constant $passed-attributes
+  = text-attributes(foreground: $color-green);
+define constant $skipped-attributes
+  = text-attributes(foreground: $color-cyan);
+define constant $not-implemented-attributes
+  = text-attributes(foreground: $color-cyan);
+define constant $failed-attributes
+  = text-attributes(foreground: $color-red);
+define constant $crashed-attributes
+  = text-attributes(foreground: $color-red);
+define constant $component-name-attributes
+  = text-attributes(intensity: $bright-intensity);
+define constant $total-attributes
+  = text-attributes(intensity: $bright-intensity);
+define constant $count-attributes
+  = text-attributes(intensity: $bright-intensity);
+
+define function result-status->attributes
```

Comment:
Using a Unicode arrow would (currently) blow things up.

I'm hoping that using a ``->`` in a function name like that catches on. :) There are a lot of places where I think it would be useful.  I like ``integer->string`` more than ``integer-to-string`` but I definitely like ``result-status->attributes`` more than ``result-status-to-attributes``.